web/src/lib/utils/focus-util.ts

```diff
@@ -29,7 +29,7 @@ export const focusNext = (selector: (element: HTMLElement | SVGElement) => boole
   const totalElements = focusElements.length;
   let i = index;
   do {
-    i = (i + (forwardDirection ? 1 : -1) + totalElements) % totalElements;
+    i = (i + (direction === 'next' ? 1 : -1) + totalElements) % totalElements;
```

Comment:
Why so complicated? lmao `(i + totalElements) % totalElements == i` for all `i`.
```suggestion
    i = (i + (direction === 'next' ? 1 : -1)) % totalElements;
```

Reply:
`totalElements` is used to handle negative numbers when moving backwards - otherwise, javascript's modulus operator returns a negative number, causing wrapping to not work.